src/arch/arm/object/vcpu.c

```diff
@@ -227,6 +200,7 @@ void vcpu_init(vcpu_t *vcpu)
     /* Virtual Timer interface */
     vcpu->virtTimer.last_pcount = 0;
 #endif
+    vcpu->vgic.vmcr = VGIC_VMCR_VPMR | VGIC_VMCR_VENG1;
```

Comment:
I'm also going to remove this initialization and leave these registers initialized to 0 as before.